src/gui/editors/PianoRoll.cpp

```diff
@@ -3530,22 +3580,6 @@ void PianoRoll::finishRecordNote(const Note & n )
 						it->key(), it->getVolume(),
 						it->getPanning() );
 				n1.quantizeLength( quantization() );
-
-				//Get selected chord
-				const InstrumentFunctionNoteStacking::Chord & chord = InstrumentFunctionNoteStacking::ChordTable::getInstance()
-					.getChordByName( m_chordModel.currentText() );
-
-				if( !chord.isEmpty() )
```

Comment:
Can you please explain why you removed this? It looks right, I just can't see it.

Reply:
Until now chords were recorded by this and the actual chord's notes were not real, so this code virtually added them to the m_pattern, knowing that they should be added since there is a real base note. 
Now every base note is played with the rest of the chord notes (notes are actually generated on and turned off when paused). A signal of midiNoteOff is emitted for each of them, triggering the record for them just like the base note.  If I'll keep this code here - the chord will be recorded twice.